python_modules/dagster-graphql/dagster_graphql/schema/inputs.py

```diff
@@ -80,6 +80,7 @@ class GraphenePipelineSelector(graphene.InputObjectType):
     repositoryName = graphene.NonNull(graphene.String)
     repositoryLocationName = graphene.NonNull(graphene.String)
     solidSelection = graphene.List(graphene.NonNull(graphene.String))
+    assetSelection = graphene.List(graphene.NonNull(GrapheneAssetKeyInput))
```

Comment:
This is cool! I can take a look through the other resolvers once this merges and make sure there aren't places where we're passing a PipelineSelector AND a list of AssetKeys as separate arguments, since they can now be sent together.